redaxo/src/core/lib/console/package/deactivate.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ */
+class rex_command_package_deactivate extends rex_console_command
+{
+    protected function configure()
+    {
+        $this->setName('package:deactivate')
+            ->setDescription(rex_i18n::msg('package_deactivate_help'))
+            ->addArgument('package-id', InputArgument::REQUIRED, rex_i18n::msg('package_id_help'));
```

Comment:
Ehrlich gesagt glaube ich, ich würde die Commands ohne i18n komplett in Englisch lassen.
Es ist ja nicht so, dass irgendwann man ein Schwedischer Redakteur oder so die console nutzen wird.

Oder wie siehst du das @staabm?

Also ich hätte es vor allem gerne einheitlich. Und finde es eigentlich unnötig, gerade für vielleicht zukünftig bisschen umfangreichere Commands, alles über i18n zu lösen. Die Grundsprache von symfony/console ist ja nun mal auch Englisch.
Vielleicht sollten wir dann in der cli sogar hartkodiert rex_i18n auf englisch stellen, sodass falls irgendwo die Messages genutzt werden, die englischen verwendet werden?

Gerne diskutieren/überlegen, bin da auch noch nicht sicher, was insgesamt der beste Weg ist!

Reply:
Kann ich gut mit leben

Reply:
>Die Grundsprache von symfony/console ist ja nun mal auch Englisch.

Das stimmt allerdings. Zudem nutzen wie @gharlan schon sagt, immer die Entwickler/Admins die console. Die sollten ja der englischen Sprache fähig sein.

Fazit: Ich kann mich mit der hartkodierten englischen Ausgabe ohne rex_i18n sehr gut anfreunden.

Reply:
Sorry fzr das hin und her aber english only hardcodiert wäre super

Reply:
np. wird gemacht :+1:

Reply:
> Die sollten ja der englischen Sprache fähig sein.

Das mag sein, aber wenn ich deutsch haben kann nehme ich lieber deutsch als englisch. Und wenn i18n bereitsteht, warum nicht nutzen?

Reply:
Ich finde wir sollten erstmal so anfangen und schauen ob die commandos überhaupt nützlich sind und genutzt werden

Reply:
Sollte rex_i18n erstmal drin bleiben? Bin ein bisschen verwirrt :laughing: